ui/app/src/views/ViewProject.tsx

```diff
@@ -48,14 +52,56 @@ function ViewProject() {
   if (projectName === undefined) {
     throw new Error('Unable to get the Project name');
   }
+
+  const navigate = useNavigate();
+
+  const [deleteProjectConfirmState, setDeleteProjectConfirmState] = useState<boolean>(false);
+  const openDeleteProjectConfirmDialog = () => {
+    setDeleteProjectConfirmState(true);
+  };
+
+  const closeDeleteProjectConfirmDialog = () => {
+    setDeleteProjectConfirmState(false);
+  };
+
+  const { successSnackbar, exceptionSnackbar } = useSnackbar();
+  const deleteProjectMutation = useDeleteProjectMutation();
+  const triggerProjectDeletion = function () {
+    deleteProjectMutation.mutate(projectName, {
+      onSuccess: (name: string) => {
+        successSnackbar(`project ${name} is successfully deleted`);
```

Comment:
```suggestion
        successSnackbar(`project ${name} was successfully deleted`);
```

Reply:
Thanks, I applied it for all other "is successfully" mentions